crates/oxc_transformer/src/es2021/logical_assignment_operators.rs

```diff
@@ -1,3 +1,59 @@
+//! ES2021: Logical Assignment Operators
+//!
+//! This plugin transform logical assignment operators `&&=`, `||=`, and `??=` to a series of logical expressions.
+//!
+//! > This plugin is included in `preset-env`, in ES2021
+//!
+//! ## Example
+//!
+//! Input:
+//! ```js
+//! a ||= b;
+//! obj.a.b ||= c;
+//!
+//! a &&= b;
+//! obj.a.b &&= c;
+//! ```
+//!
+//! Output:
+//! ```js
+//! var _obj$a, _obj$a2;
+//!
+//! a || (a = b);
+//! (_obj$a = obj.a).b || (_obj$a.b = c);
+//!
+//! a && (a = b);
+//! (_obj$a2 = obj.a).b && (_obj$a2.b = c);
+//! ```
+//!
+//! ### With Nullish Coalescing
+//!
+//! > While using the [nullish-coalescing-operator](https://github.com/oxc-project/oxc/blob/main/crates/oxc_transformer/src/es2020/nullish_coalescing_operator.rs) plugin (included in `preset-env``)
+//!
+//! Input:
+//! ```js
+//! a ??= b;
+//! obj.a.b ??= c;
+//! ```
+//!
+//! Output:
+//! ```js
+//! var _obj$a, _obj$a2;
+//! var _a, _obj$a, _obj$a$b;
```

Comment:
I think only the 2nd line should be here.
```suggestion
//! var _a, _obj$a, _obj$a$b;
```